### ISSUE TITLE
[Feature]: Learn about PyTorch & Neural Networks

### ISSUE THREAD

**Issue by Mikatech** [PoCInnovation/SpaceVector#2]
### Description

Complete the POC & 42AI technical exercise to gain skills in neural networks.

### Expected behaviour

The jupyter notebook must be completed and documented on the methods used.
The students must be able to explain the basic functioning of a neural network.

- [x] The exercise is successful
- [x] The Notebook is documented
- [x] Visualization plots on loss and accuracy have been developed

### Environment

/

### Proposed solution

/
